lib/ui/compositing/scene_builder.h

```diff
@@ -109,15 +106,16 @@ class SceneBuilder : public RefCountedDartWrappable<SceneBuilder> {
  private:
   SceneBuilder();
 
-  std::shared_ptr<flutter::ContainerLayer> root_layer_;
-  flutter::ContainerLayer* current_layer_ = nullptr;
+  bool has_layer() const { return !layer_stack_.empty(); };
```

Comment:
It looks like this is never used.  The `SceneBuilder` constructor adds a root layer that is never popped, so the layer stack should always be nonempty.

Reply:
Oops, good catch.  this was left over from an earlier iteration of the code.